### DISCUSSION TITLE
Acmebot Error

### DISCUSSION THREAD

**Discussion by ChicagoDave** [polymind-inc/acmebot#205] [Q&A] [answered]
Getting FinalizedOrder error - unsure what's missing.

![image](https://user-images.githubusercontent.com/1648343/102567648-8cbf5e80-40a7-11eb-8d33-8b8ee68c5f55.png)

**Comment by ChicagoDave** [comment]
Also the popup window is locked...you can't copy any of the text.

**Comment by shibayan** [comment, marked as answer]
Since it seems to be an authentication error, please make sure that the Key Vault access policy settings are correct.

https://github.com/shibayan/keyvault-acmebot#5-add-to-key-vault-access-policies-if-you-use-an-existing-key-vault

**Reply by ChicagoDave** [reply]
Yeah I missed some basic Azure tasks while doing this....I have it all working now. Admittedly, the documentation here assumes you're a strong Azure DevOps person. If I have time, I may write a blog about my experience with Acmebot.
